<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gradeManage</class>
 <widget class="QWidget" name="gradeManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1105</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>成绩信息管理</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="gradeTable">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>751</width>
     <height>401</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>学号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>课程号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>课程名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>得分</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>任课教师</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>130</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>学号：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="sId_lineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>课程号：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="cId_lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>200</y>
     <width>222</width>
     <height>65</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="searchBysNo">
      <property name="text">
       <string>根据学号查找</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="searchBycId">
      <property name="text">
       <string>根据课程号查找</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="listAll_button">
      <property name="text">
       <string>查看所有成绩</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="addGrade_button">
      <property name="text">
       <string>添加成绩信息</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>280</y>
     <width>267</width>
     <height>65</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="1">
     <widget class="QPushButton" name="delete_button">
      <property name="text">
       <string>删除选中成绩信息</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="student_statistics">
      <property name="text">
       <string>学生个人成绩统计</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="course_statistics">
      <property name="text">
       <string>课程成绩统计</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="update_gradeButton">
      <property name="text">
       <string>修改选中成绩信息</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="currentSelected_label">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>100</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>当前选中：</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>370</y>
     <width>196</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QPushButton" name="export_button">
      <property name="text">
       <string>导出为Excel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="back_button">
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
